context/src/main/java/io/opentelemetry/context/PersistentHashArrayMappedTrie.java

```diff
@@ -3,6 +3,23 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+// Includes work from:
```

Comment:
```suggestion
// The code in this file was developed by:
```

Or something else to indicate that this file includes *solely* (mostly) work from gRPC might be better.

Reply:
What will be the criterium for "mostly"? When should we use one variant or another?

Reply:
Yeah I think it's a bit hard to judge, and we may modify the file later. In my experience, I've seen dual-license applied to files that are completely copied, not sure but probably for these reasons.

Reply:
I think just keeping the "Includes work from" is a good all-encompassing bit of verbiage.